profiles/epub.md

```diff
@@ -53,6 +53,7 @@ This profile defines additional Link properties:
 | ----- | --------- | -------- | --------- | 
 | [contains](#contains)  | Identifies content contained in the linked resource, that cannot be strictly identified using a media type.  | Array  | `mathml`, `onix`, `remote-resources`, `js`, `svg` or `xmp`  | 
 | [layout](#layout)  | Hint about the nature of the layout for the linked resources.  | String  | `fixed` or `reflowable`  | 
+| [page](#page)  | Indicates how the linked resource should be displayed in a reading environment that displays synthetic spreads. | String  | `right`, `left` or `center`  | 
```

Comment:
Isn't `page` also useful for Divina? I thought it would be part of the core Link properties.

Reply:
Yeah that makes sense, we would define it in https://readium.org/webpub-manifest/properties.html in this case.